<launch>
<!-- Launch file for Livox AVIA LiDAR -->

  <arg name="rviz" default="true" />
  <rosparam command="load" file="$(find std_detector)/config/config_ntu.yaml" />
  <node  pkg="std_detector" type="relocalization_ntu" name="relocalization_ntu" output="screen"/>

  <!-- Pose gt -->
  <param name="reference_gt_path"            type="string" value = ""/>

  <param name="localization_lidar_path"      type="string" value = "/home/dat/data/std_reloc/ntu_day_02/KFCloud/"/>
  <param name="localization_pose_path"       type="string" value = "/home/dat/data/std_reloc/ntu_day_02/KfCloudPose.pcd"/>
  
  <param name="generate_descriptor_database" type="bool"   value = "false"/>
  
  <param name="generated_lidar_path"         type="string" value = "/home/dat/data/std_reloc/ntu_day_01/KFCloud/"/>
  <param name="generated_pose_path"          type="string" value = "/home/dat/data/std_reloc/ntu_day_01/kfpose_fixed.pcd"/>
  <param name="descriptor_file_path"         type="string" value = "/home/dat/data/std_reloc/priormap/saved_descriptors/"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find std_detector)/rviz_cfg/demo.rviz" />
  </group>

</launch>